Source/ThirdParty/tweetnacl/randombytes.cpp

```diff
@@ -0,0 +1,16 @@
+#include <random>
+#include <cmath>
+
+extern "C"
+{
+
+void randombytes(unsigned char *x,unsigned long long xlen)
+{
+    std::mt19937 rng(std::random_device{}());
```

Comment:
Probably should be global. Depending on how it initializes we may end up either with same state or with very predictable random numbers. Does it not need seeding?

Reply:
It seeds from random device.